proto/sentry_protos/snuba/v1/endpoint_find_traces.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "google/protobuf/timestamp.proto";
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+message FindTracesRequest {
```

Comment:
What will the sort order of the response be? The ask is to have it sortable by timestamp or trace duration, with timestamp being the more common sort.

Reply:
I'll add something to change the sort order. This API spec makes no distinction on how the results are sorted, so we can do that when we actually implement the endpoint.